eo-phi-normalizer/app/Main.hs

```diff
@@ -147,6 +150,9 @@ dependenciesArg =
 jsonSwitch :: Parser Bool
 jsonSwitch = switch (long "json" <> short 'j' <> help "Output JSON.")
 
+latexSwitch :: Parser Bool
+latexSwitch = switch (long "latex" <> short 'l' <> help "Output LaTeX.")
```

Comment:
I think this one does not need a short name. And `--tex` is better than `--latex`:
```suggestion
latexSwitch = switch (long "tex" <> help "Output LaTeX.")
```